web/app/themes/ppj/partials/videoPlayer.php

```diff
@@ -2,24 +2,25 @@
 
 global $ppj_template_data;
 $td = $ppj_template_data;
+$coverImage = wp_get_attachment_image(
+    $td['cover-image-id'],
+    'large',
+    false,
+    [
+        'class' => 'video-player__cover-image',
+        'sizes' => '(min-width: 1440px) 792px, (min-width: 1024px) 915px, (max-width: 768px) calc((100vw - 64px) * 0.75), 0.85vw'
+    ]
+);
+$coverImageURL = wp_get_attachment_image_url($td['cover-image-id'], 'large');
 
 ?>
 
 <div class="video-player video-player--<?= $td['host'] ?>"
      id="<?= $td['id'] ?>"
 >
     <?php if ($td['cover-image-id']): ?>
-        <a href="#" class="video-player__cover" aria-label="Play Video" data-ie-bg-img="url('<?= $td['cover-image'] ?>')">
-            <?php
-
-            echo wp_get_attachment_image(
-                $td['cover-image-id'], 'large', false, [
-                    'class' => 'video-player__cover-image',
-                    'sizes' => '(min-width: 1440px) 792px, (min-width: 1024px) 915px, (max-width: 768px) calc((100vw - 64px) * 0.75), 0.85vw'
-                ]);
-
-            ?>
-
+        <a href="#" class="video-player__cover" aria-label="Play Video" data-bg-img-url="url('<?= $coverImageURL ?>')">
```

Comment:
1. I'd recommend you use the `esc_attr` function here to escape the URL, since you're putting it inside an attribute. It's unlikely that the string will contain `"` double quotes, but it's good practice nonetheless.

2. Also I'm not sure if the `url('')` part is necessary inside this attribute – couldn't that just be applied in the JavaScript, rather than repeated throughout all applicable attributes? That way the attribute would simply contain the URL to the background image, as its name suggests.

Reply:
1 & 2 done.